dask/array/chunk.py

```diff
@@ -1,248 +1,295 @@
-""" A set of NumPy functions to apply per chunk """
```

Comment:
Looks like a github bug getting confused by DOS/Unix line terminations? The commit looks right both in PyCharm and in Gitkraken...